src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -413,6 +440,16 @@ static void _repeat(LottieGroup* parent, unique_ptr<Shape> path, RenderContext*
         mathTranslateR(&m, -repeater->anchor.x, -repeater->anchor.y);
 
         auto pm = PP(shape)->transform();
+
+        if (repeater->transform && !mathIdentity(static_cast<const Matrix*>(repeater->transform))) {
```

Comment:
You might wanna check the mathIdentity == true case. I assume its much less than false scenarios. If so, it hurts the bigger case, it shouldn't be.

Reply:
The issue is that at a given level, we already have an accumulated transformation matrix, and we're accumulating that before the repeater appears. I'll think about it some more.

Reply:
Tip: simply print out each case tested with Lottie and see which case prints more.

Reply:
not sure any more if I understand - yes, often this won't be an identity matrix. so I removed the check. is that what you meant?

Reply:
Checking the identity matrix also comes with a comparison cost. If the identity matrix is used more frequently compared to the non-identity matrix, we can keep it; otherwise, I'd suggest removing it. I think you can' briefly estimate this with `Lottie` example.